websites/C/Currys/presence.ts

```diff
@@ -0,0 +1,91 @@
+const presence = new Presence({
+		clientId: "996516483150663721",
+	}),
+	{ pathname, host } = document.location,
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData = {
+		smallImageKey: "search",
+		largeImageKey: "https://i.imgur.com/yOolF4f.png",
+		largeImageText: "Browsing Currys",
+		startTimestamp: browsingTimestamp,
+		details: "Browsing",
+		state: "Currys Website",
+	};
+	if (host === "www.currys.co.uk") {
```

Comment:
Remove this. Due to the metadata file it'll only load on currys.co.uk or www.currys.co.uk

Reply:
Done.